### PR TITLE
Modal and various style fix

### PR DESCRIPTION
**Pull request by domoritz**
Fixes https://github.com/vega/voyager/issues/215

### CODE REVIEW

**Comment by kanitw** [comment]
Now ready for review with working scrollbar — please see commit logs for detail

File: src/app/main/main.html [hunk 1]
```diff
@@ -63,11 +63,11 @@
     </div>
   </div>
 
-  <modal id="bookmark-list" ng-if="Bookmarks.isSupported">
```
Comment (domoritz) [inline src/app/main/main.html]: have you tested the embedded mode (load in iframe with `sandbox="allow-scripts"`?

**Comment by domoritz** [comment]
LGTM except for the sandboxing comment.

File: src/app/main/main.html (hunk 1 above)
Comment (domoritz) [inline src/app/main/main.html]: Whatever, I'll fix that later.

**Pull request merged**